docker-entrypoint.sh

```diff
@@ -168,6 +168,10 @@ if expr "$1" : "apache" 1>/dev/null || [ "$1" = "php-fpm" ] || [ "${NEXTCLOUD_UP
                             NC_TRUSTED_DOMAIN_IDX=$(($NC_TRUSTED_DOMAIN_IDX+1))
                         done
                     fi
+
+                    if [ -n "${NEXTCLOUD_CUSTOM_INSTALL_SCRIPT+x}" ]; then
+                        run_as "/bin/sh ${NEXTCLOUD_CUSTOM_INSTALL_SCRIPT}"
```

Comment:
I'm fine with this but are there any common use cases where you don't want to `run_as` limited user?

Reply:
The only use case I see may be adding distrib packages, but I think that building a custom image is better here.